### PR TITLE
[ISSUE #3900] Removed Additional Null Check

### LINKED ISSUE #3900: [Enhancement] Nullcheck of value previously dereferenced
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/catalog/protos/Operation.java line 87-89
analysis and explanation:
A value is checked here to see whether it is null, but this value can't be null because it was previously dereferenced and if it were null a null pointer exception would have occurred at the earlier dereference. Essentially, this code and the previous dereference disagree as to whether this value is allowed to be null. Either the check is redundant or the previous dereference is erroneous.

### Describe the solution you'd like

Remove redundant NPE checks.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by Ishaan29**
ISSUE #3900
Removed additional null check as both Objects.requireNonNull and later explicit null check is doing the same thing.

Fixes #3900.

### Motivation

* as the issue suggest, there is an extra null check that is exhibiting the same functionality. as the Objects.requireNonNull throws NPE if the object is null. the later null check is redundant the execution won't reach the later check. 

### Modifications

* Removed the additional redundant null check. 

### Documentation

- Does this pull request introduce a new feature? (yes / no) no
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why? simple code refactoring.
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by Alonexc — APPROVED** [review]
LGTM

**Comment by Ishaan29** [comment]
hey @xwm1992 ,

could you please review this ?

**Review by mxsm — APPROVED** [review]
LGTM~

**Comment by Ishaan29** [comment]
hey @Alonexc @mxsm , 

as I am new here, I am not sure as to how should we go about merging the request, 
could you please let me know if we need more approvals, or essentially what are the nest steps. 

thanks

**Comment by Alonexc** [comment]
@Ishaan29 Once the ci check is passed, it can be merged in.

**Comment by Ishaan29** [comment]
hey @Alonexc , 
how do we trigger CI ?

**Review by xwm1992 — APPROVED** [review]

**Review by lrhkobe — APPROVED** [review]

**Pull request merged**
